examples/scripts/script_reslc.py

```diff
@@ -0,0 +1,143 @@
+"""Example script of calculating SLC from interferograms.
+
+This .py script is designed to be executed with a Dask SLURMCluster on a SLURM HPC.
+It should be executed through a SLURM script by `sbatch` command.
+Please do not run this script by "python xxx.py" on a login node.
+"""
+
+import logging
+import os
+import socket
+from pathlib import Path
+import numpy as np
+from matplotlib import pyplot as plt
+from dask.distributed import Client
+from dask_jobqueue import SLURMCluster
+import sarxarray
+
+from pydepsi.slc import intf_to_slc
+from pydepsi.io import read_metadata
+
+# Make a logger to log the stages of processing
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+ch = logging.StreamHandler()  # create console handler
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
+
+
+def get_free_port():
+    """Get a non-occupied port number."""
+    sock = socket.socket()
+    sock.bind(("", 0))  # Bind a port, it will be busy now
+    freesock = sock.getsockname()[1]  # get the port number
+    sock.close()  # Free the port, so it can be used later
+    return freesock
+
+
+# ---- Config 1: Path Configuration ----
+# Paths and files
+stack_dir = Path("/project/caroline/Share/stacks/nl_veenweiden_s1_dsc_t037/stack")  # Intf stack dir
+mother_dir = stack_dir / "20200328"  # Mother image dir
+reading_chunks = (2000, 2000)  # Reading chunks from binary
+
+# Output config
+overwrite_zarr = False  # Flag for zarr overwrite
+writing_chunks = (2000, 2000)  # Writing chunks to zarr, (azimuth, range)
+path_stm = Path("./stm.zarr")  # Zarr output storage for STM
```

Comment:
?
No stm involved here. Parameter is not used. Remove.

Reply:
```suggestion
```